charmcraft/application/commands/lifecycle.py

```diff
@@ -161,26 +160,6 @@ def _validate_bases_indices(self, bases_indices):
             if bases_index >= len_configured_bases:
                 raise CraftError(msg.format(bases_index))
 
-    def run_managed(self, parsed_args: argparse.Namespace) -> bool:
```

Comment:
This technically changes things on Linux, but it's irrelevant because we're going to be eliminating bundles in #1858